clang-tools-extra/clangd/3CCommands.cpp

```diff
@@ -48,14 +48,17 @@ void as3CCommands(const Diagnostic &D, std::vector<Command> &OutCommands) {
 }
 
 bool is3CCommand(const ExecuteCommandParams &Params) {
-  return (Params.command.rfind(std::string(Command::_3C_APPLY_ONLY_FOR_THIS), 0) == 0) ||
-         (Params.command.rfind(std::string(Command::_3C_APPLY_FOR_ALL), 0) == 0);
+  return (Params.command.rfind(std::string(Command::_3C_APPLY_ONLY_FOR_THIS),
+                               0) == 0) ||
+         (Params.command.rfind(std::string(Command::_3C_APPLY_FOR_ALL), 0) ==
+          0);
 }
```

Comment:
This is a shame. Harder to read.

Reply:
Agreed, but I assume `clang-format` is applying its standard wrapping rules based on the structure of the expression and I don't think it's worth fighting at this time, especially in `clangd3c`.  If we cared, we could probably find a way to refactor the code that would result in more reasonable formatting.